CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-03-31
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* Force migrating all Image Cleaner clusters to addon v2.
+
+### Release Notes
+* Features:
+  * Created ZRS disk by default in built-in storage classes on a multi-zone AKS cluster
+
+* Component Updates: 
+  * Bump csi driver to v1.4.2.
```

Comment:
which one? Files/disks/blob? And we need to include link to the release notes of each of those

Reply:
@andyzhangx , do you have the link for the release?

Reply:
https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases/tag/v1.4.2

Reply:
Thanks, updated!